### ISSUE TITLE
Is there endpoint for GET followers/list?

### ISSUE THREAD

**Issue by githubbar** [DocNow/twarc#637]

**Comment by igorbrigadir** [comment]
No, by default, twarc uses https://developer.twitter.com/en/docs/twitter-api/v1/accounts-and-users/follow-search-get-users/api-reference/get-friends-ids for v1.1 API. It might be useful to have this endpoint too, some cases may benefit from calling this endpoint (as you can increase your rate limits by using this one and `/ids` together). It will be worth implementing this in the client library, but not in the CLI, in my opinion.

For v2 API we use https://developer.twitter.com/en/docs/twitter-api/users/follows/api-reference
